[custom keyword] Update doc to reflect current training time (#132)
MicrosoftDocs/azure-ai-docs: articles/ai-services/speech-service/includes/quickstarts/keyword-recognition/use-speech-studio.md

1. Select a model type, then select **Create**. You can view a list of regions that support the **Advanced** model type in the [Keyword recognition region support](../../../regions.md#speech-service) documentation. 

> 1. Due to high demand, training the basic model may take several hours, while training the advanced model could take up to a day to complete. The status will update from **Processing** to **Succeeded** once the training is complete. 

Reviewer: ```suggestion
1. Because of high demand, training the basic model might take several hours. Training the advanced model could take up to a day to finish. The status changes from **Processing** to **Succeeded** when the training is complete. 
```
Edits to align with Microsoft style
